### PR TITLE
Security series webinar

### LINKED ISSUE #8184: [Event + takeover] New Security Series Webinar - Build the Takeover +  Event page
## Requirements

Please build a takeover and an engage page based on this [brief](https://docs.google.com/document/d/1ZEqDp0Mj8LOttMG5YyPcy5jAatrhGK1IgO0AVFEp6G0/edit) and [design](https://github.com/canonical-web-and-design/web-squad/issues/3114):
 
- [x] build takeover
- [x] build event page at /security-series
- [x] add takeover to `/takeovers/index.html`
- [x] add takeover to `/index.html`
- [x] build a [social banner](https://tools.demos.haus/) and use the 1200px x 630px one
- [ ] test the page on [https://socialdebug.com/](https://socialdebug.com/) or [http://debug.iframely.com/](http://debug.iframely.com/) and [https://cards-dev.twitter.com/validator](https://cards-dev.twitter.com/validator)

### PR DESCRIPTION
**Pull request by bethcollins92**
## Done

- Build takeover
- Build even page at `/security-series`
- Add takeover to `takeovers/index.html`
- Add takeover to `index.html`
- Build [social banner](https://tools.demos.haus/?title=Ubuntu+Security%3A+Security+and+compliance&subtitle=The+full+Ubuntu+security+story.+Learn+how+our+teams+are+securing+all+your+open+source%2C+from+cloud+to+edge.&illustration_url=https%3A%2F%2Fassets.ubuntu.com%2Fv1%2F1899df43-ubuntu-advantage-circular_white.svg&background=grad#banner-form-modal)

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/security-series
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check the site matches the [design](https://user-images.githubusercontent.com/65166314/93861516-ed194c80-fcc0-11ea-933e-a87dca1a7375.png) and the [brief](https://docs.google.com/document/d/1A7JDBRZ_tuEvteE1pVA9_rV-oJUq1_DuT7GmO6zVCXM/edit#)
- Check takeover matches [design](https://github.com/canonical-web-and-design/web-squad/issues/3114) and [brief](https://docs.google.com/document/d/1duufuOPuy6asv-i9qtGNs3y1VLRO67GYaOfwE9OGQmw/edit)

## Issue / Card

Fixes #8184 

## Screenshots

image

### CODE REVIEW

**Comment by pmahnke** [comment]
@Khutch25 why not 'Register for the webinar' ?  I know you know they are all webinars, but will someone landing on the page from an ad?

File: templates/security-series/index.html [hunk 1]
```diff
@@ -0,0 +1,449 @@
...
+        ) | safe
+      }}      
+      </p>   
+      <p class="p-heading--4">Securing open source across multi-cloud environments</p>
+      <p>Join this webinar to see how you can ensure performant open source in production environments and achieve maximum availability by reducing downtime and providing access to high and critical CVE fixes. </p>
+      <p>Furthermore, learn how Ubuntu helps organisations remain compliant with government and industry standards and regulations, including Common Criteria EAL2 with FIPS 140-2 Level 1 certified crypto modules.</p>
+      <a href="https://www.brighttalk.com/webcast/6793/440517">Resister</a>
+    </div>
+    <div class="col-6">
+      <p>
```
Comment (pmahnke) [inline templates/security-series/index.html:366]: some of the space around the illustrations is off

image


some are in `p` tags and some have `u-sv1` can you check them?

**Review by pmahnke — CHANGES_REQUESTED** [review]
A few comments inline.

**Comment by Khutch25** [comment]
> @Khutch25 why not 'Register for the webinar' ? I know you know they are all webinars, but will someone landing on the page from an ad?

Yes, you are right. Is it too late to align them all to 'Register for webinar' with the offsite/new tab link icon? 

I've made updates to the copy doc to reflect this - https://docs.google.com/document/d/1A7JDBRZ_tuEvteE1pVA9_rV-oJUq1_DuT7GmO6zVCXM/edit

File: templates/security-series/index.html [hunk 2]
```diff
@@ -0,0 +1,449 @@
...
+    <div class="col-4">
+      <p>
+      {{
+        image(
+         url="https://assets.ubuntu.com/v1/1d5f293c-compliance.svg",
+         alt="",
+         height="80",
+         width="130",
+         hi_def=True,
+         attrs={"class": ""},
```
[the remark this answers is not in the document]
Comment (bethcollins92) [reply templates/security-series/index.html:101]: Spoken to @meralonso and she said these illustrations are going to be videos in the future so they can't be hidden on small screens

**Comment by bethcollins92** [comment]
Ready for another review

File: templates/security-series/index.html [hunk 3]
```diff
@@ -0,0 +1,449 @@
...
+          loading="auto",
+        ) | safe
+      }}      
+      </p>
+      <p class="p-heading--4">Securing open source across multi-cloud environments</p>
+      <p>Join this webinar to see how you can ensure performant open source in production environments and achieve maximum availability by reducing downtime and providing access to high and critical CVE fixes.</p>
+      <p>Furthermore, learn how Ubuntu helps organisations remain compliant with government and industry standards and regulations, including Common Criteria EAL2 with FIPS 140-2 Level 1 certified crypto modules.</p>
+      <a class="p-link--external" href="https://www.brighttalk.com/webcast/6793/440517">Register for the webinar</a>
+    </div>
+    <div class="col-6">
```
Comment (pmahnke) [inline templates/security-series/index.html:177]: you need a bit of space between this and the next row
```suggestion
    <div class="col-6 u-sv3">
```

**Review by pmahnke — APPROVED** [review]
one last comment then good to go

**Comment by pmahnke** [comment]
@bethcollins92 can you also take down the telco virtual event takeover with this? cc @SirSamTumless

**Comment by meralonso** [comment]
Hey @bethcollins92 I'm happy with the layout but all illustrations are pixelated for some reason - could you please have a look into that? 

Screenshot 2020-09-29 at 15 16 11

Screenshot 2020-09-29 at 15 20 11

**Comment by meralonso** [comment]
> Hey @bethcollins92 I'm happy with the layout but all illustrations are pixelated for some reason - could you please have a look into that?
> 
> Screenshot 2020-09-29 at 15 16 11
> 
> Screenshot 2020-09-29 at 15 20 11

@bethcollins92 according to @sowasred2012 the pixelated images seems to be a different issue and he's going to flag that up

**Comment by anthonydillon** [comment]
@bethcollins92 Can you reset the commits in this PR and ask Steve for help force pushing to master to fix the history and propose a new PR with cherry picked commits please.

**Pull request merged**